index.html

```diff
@@ -1,41 +1,291 @@
+    </section>
 
-    </header>    
-    <section id="home_content">
-        <h1>I'm Gerg
-         Glad to see you!</h1>
-         <p>
-             I'm a remote software developer! I can help you build a product , 
-             feature or website Look through some of my work and experience! 
-             If you like what you see and have a project you need coded, don't hestiate to contact me.
-         </p>
-         <footer>
-             <h2>LET'S CONNECT</h2>
-             <div id="contact">
-                <ul>
-                  <li class="icon"><a href="#"><img src="img/twitter.png" alt="twitter link"></a></li>
-                  <li class="icon"><a href="#"><img src="img/Vector.png" alt="linked link"></a></li>
-                  <li class="icon"><a href="#"><img src="img/medium.png" alt="M"></a> </li>
-                  <li class="icon"><a href="#"><img src="img/github.png" alt="github link"></a></li>
-                  <li class="icon"><a href="#"><img src="img/angellist.png" alt="doit"></a></li>
-                </ul>
-             </div>
-         </footer>
-     </section>
+    <section class="works" id="works">
+      <ul class="cards ul grid">
+        <li class="card flex background-white">
+          <img class="card__img" src="img/tonic-project.png" alt="tonic project" />
+          <div class="card__text">
+            <h2 class="card__title weight-700 size-32 line_hieght-44 clr-n-800">
+              Tonic
+            </h2>
+            <div class="card__subtitle flex">
+              <p class="card__subtitle-1 weight-600 size-13 line_hieght-16 clr-n-600">
+                CANOPY
+              </p>
+              <ul class="card__subtitle-ul flex">
+                <li class="card__subtitle-li weight-600 size-13 line_hieght-16 clr-n-500">
+                  Back End Dev
+                </li>
+                <li class="card__subtitle-li weight-600 size-13 line_hieght-16 clr-n-500">
+                  2015
+                </li>
+              </ul>
+            </div>
+            <p class="card__dsc weight-400 size-15 line_hieght-24 clr-n-600">
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required.
+            </p>
+            <div class="card__topics flex">
+              <ul class="langs">
+                <li><p class="card__topics__text weight-500 size-12 line_hieght-16 clr-p-400 bg-p-50">html</p></li>
+                <li><p class="card__topics__text weight-500 size-12 line_hieght-16 clr-p-400 bg-p-50">css</p></li>
+                <li><p class="card__topics__text weight-500 size-12 line_hieght-16 clr-p-400 bg-p-50">javascript</p></li>
+              </ul> 
+            </div>
+            <div class="card__btn">
+              <button type="button" class="btn weight-500 size-17 line_hieght-24 clr-p-500 background-white first-btn" id="0">
+                See Project
+              </button>
+            </div>
+          </div>
+        </li>
```

Comment:
- [ ] I notice your work card is a bit different from the Figma template. You are expected to stick to the template as much as possible, this includes fonts, colors, layouts, spacings, and sizes. Kindly consider the following;
  - Consider changing the bullet type to match the template.
  - Consider reducing the font weight of the description.
  - Consider removing the bullets on the list of technologies and lay them out to match the Figma template. I suggest you use flexbox to lay out the technologies.
### Your design 
![image](https://user-images.githubusercontent.com/94551484/227543827-a41a35e6-222f-41ef-8b04-6d9078cf47fa.png)
### Figma
![image](https://user-images.githubusercontent.com/94551484/227544042-a080bd58-fcea-4520-8afc-9e7506cfdaff.png)